server/ServerApp.h

```diff
@@ -307,6 +307,9 @@ class ServerApp final : public IApp {
     /** Called when asyncio timer ends. Executes Python asyncio callbacks if any was generated. */
     void AsyncIOTimedoutHandler(const boost::system::error_code& error);
 
+    /** Called when new \a turn state received by player playing \a empire_id. */
+    void UpdateEmpireTurnReceived(int empire_id, int turn, bool success);
```

Comment:
this is declared but not defined?

Reply:
Put definition.

Reply:
never called though...

Reply:
I'll add it later either with callbacks or signals.